includes/class-wporg-gp-translation-events-translation-listener.php

```diff
@@ -157,10 +157,11 @@ function ( $event ) use ( $at ) {
 	// phpcs:disable Generic.CodeAnalysis.UnusedFunctionParameter.FoundAfterLastUsed
 	// phpcs:disable Generic.CodeAnalysis.UnusedFunctionParameter.Found
 	private function select_events_user_is_registered_for( array $events, int $user_id ): array {
+		$attending_event_ids = get_user_meta( $user_id, WPORG_GP_Translation_Events_Route::USER_META_KEY_ATTENDING, true );
 		return array_filter(
 			$events,
 			function ( $event ) use ( $attending_event_ids ) {
```

Comment:
@amieiro in #60 you removed the `WPORG_GP_Translation_Events_Event` type hint. @trymebytes you added the `id()` below. So this means this is a `WPORG_GP_Translation_Events_Event` after all and we could add the type hint back?